[new-product] Quasar (#3578)
captn3m0/endoflife.date: products/quasar.md

---
title: Quasar
category: framework
tags: javascript-runtime
iconSlug: quasar
permalink: /quasar
versionCommand: quasar -v
releasePolicyLink: https://github.com/quasarframework/quasar/blob/dev/ROADMAP.md#support-policy-and-schedule
changelogTemplate: https://github.com/quasarframework/quasar/releases/tag/quasar-v__LATEST__

> activeSupportColumn: true

Reviewer: ```suggestion
activeSupportColumn: true
eolColumn: LTS support
```

So that the `Security support` column has the same name as in the Quasar documentation.

releaseDateColumn: true

auto:
-   npm: quasar

identifiers:
-   purl: pkg:npm/quasar

releases:
-   releaseCycle: "2"
    support: true
    eol: false
    latest: "2.12.7"

>     lts: false

Reviewer: ```suggestion
```

Already the default value.

    latestReleaseDate: 2023-09-13

>     releaseDate: 2021-07-21

Reviewer: ```suggestion
    releaseDate: 2021-06-21
```

According to https://github.com/quasarframework/quasar/blob/dev/ROADMAP.md#support-policy-and-schedule.

-   releaseCycle: "1"
    support: 2021-04-01
    eol: 2023-06-30
    latest: "1.22.10"

>     lts: false

Reviewer: ```suggestion
```

Already the default value.

>     lts: false

Reviewer: ```suggestion
```

Looks like you missed this one ;).

    latestReleaseDate: 2023-08-09
    releaseDate: 2019-03-07

---

> > [Quasar](https://quasar.dev/)  is an MIT licensed open-source Vue.js based framework, which allows you as a web developer to quickly create responsive++ websites/apps in many flavours.

Reviewer: ```suggestion
> [Quasar](https://quasar.dev/) is an MIT licensed open-source Vue.js based framework, which allows web developer to quickly create responsive websites/apps in many flavors: Single Page Apps, SSR Apps, PWAs, Browser extensions, Hybrid Mobile Apps and Electron Apps.
```

> All major releases are typically supported for at least 12 months. LTS support ends 12 months after the release of a new major version.

Reviewer: Would be great to explain if possible what implies active support and what implies LTS support (for example LTS support => security patch only).

Reviewer: Did not found anything about that, let's forget it !

---
title: Quasar
category: framework
tags: javascript-runtime
iconSlug: quasar
permalink: /quasar
versionCommand: quasar -v
releasePolicyLink: https://github.com/quasarframework/quasar/blob/dev/ROADMAP.md#support-policy-and-schedule
changelogTemplate: https://github.com/quasarframework/quasar/releases/tag/quasar-v__LATEST__
releaseDateColumn: true
activeSupportColumn: true
eolColumn: LTS support

auto:
-   npm: quasar

identifiers:
-   purl: pkg:npm/quasar

> releases:

Reviewer: ```suggestion
# Support and EOL dates can be found on https://github.com/quasarframework/quasar/blob/dev/ROADMAP.md#support-policy-and-schedule.
releases:
```

To help future maintainers.

-   releaseCycle: "2"
    releaseDate: 2021-06-21
    support: true
    eol: false
    latest: "2.12.7"
    latestReleaseDate: 2023-09-13

-   releaseCycle: "1"
    releaseDate: 2019-03-07
    support: 2021-04-01
    eol: 2023-06-30
    latest: "1.22.10"

>     latestReleaseDate: 2023-08-09

Reviewer: ```suggestion
    latestReleaseDate: 2023-05-05
```

According to https://www.npmjs.com/package/quasar/v/1.22.10.
